view-user/src/pages/test/index.tsx

```diff
@@ -1,80 +1,37 @@
-import React, { useState } from "react";
-import {
-  NumberCardSmall,
-  PrizeCard,
-  NumberCardList,
-  ReactionStampModal,
-} from "@/components/common";
-import { BingoNumber } from "@/type/common";
+import React from "react";
+import Button from "@/components/common/Button";
```

Comment:
```suggestion
import { Button } from "@/components/common";
```
このようにして呼びだすように整えてるはず